web-scraper/README.md

```diff
@@ -273,247 +273,248 @@ see <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/S
-  
-  Note that each snapshot overwrites the previous one and the `saveSnapshot()`
-  calls are throttled to at most one call in two seconds,
-  in order to avoid excess consumption of resources and slowdown of the actor.
-  
-- ##### **`setValue(key, data, options): AsyncFunction`**
-
-  Sets a value to the default key-value store associated with the actor run.
-  The key-value store is useful for persisting named data records, such as state objects, files, etc.
-  The function is very similar to <a href="https://sdk.apify.com/docs/api/apify#setvalue" target="_blank"><code>Apify.setValue()</code></a>
-  function in Apify SDK.
-
-  To get the value, use the dual function `context.getValue(key)`.
-  
-  Example:
-
-  ```javascript
-  await context.setValue('my-key', { hello: 'world' });
-  ```
-  
-- ##### **`skipLinks(): AsyncFunction`**
-
-  Calling this function ensures that page links from the current page
-  will not be added to the request queue, even if they match the [**Link selector**](#link-selector)
-  and/or [**Pseudo-URLs**](#pseudo-urls) settings.
-  This is useful to programmatically stop recursive crawling,
-  e.g. if you know there are no more interesting links on the current page to follow.
-
-- ##### **`underscoreJs: Object`**
-
-  A reference to the <a href="https://underscorejs.org/" target="_blank">Underscore.js</a> library,
-  which provides various useful utility functions.
-  This field is only available if the **Inject Underscore.js** option is enabled.
-  
-  Typically, the Underscore.js object is registered under a global variable called <code>_</code>.
-  However, the web page might use this global variable for something else.
-  To avoid conflicts, the Underscore.js object is not registered globally
-  and is only available through the `context.underscoreJs` property.
-  
-  Example:
-
-  ```javascript
-  const _ = context.underscoreJs;
-  const text = _.escape('<strong>Tango & Cash</strong>');
-  ```
-
-- ##### **`waitFor(task, options): AsyncFunction`**
-
-  A helper function that waits either a specific amount of time (in milliseconds),
-  for an element specified using a CSS selector to appear in the DOM
-  or for a provided function to return `true`.
-  This is useful for extracting data from web pages with a dynamic content,
-  where the content might not be available at the time when page function is called.
-  
-  The `options` parameter is an object with the following properties and default values:
-
-  ```javascript
-  {
-    // Maximum time to wait
-    timeoutMillis: 20000,
-  
-    // How often to check if the condition changes
-    pollingIntervalMillis: 50,
-  }
-  ```
-  
-  Example:
-
-  ```javascript
-  // Wait for selector
-  await context.waitFor('.foo');
-  // Wait for 1 second
-  await context.waitFor(1000);
-  // Wait for predicate
-  await context.waitFor(() => !!document.querySelector('.foo'), { timeoutMillis: 5000 });
-  ```
+-   ##### **`customData: Object`**
```

Comment:
What happened here? Hard to see any changes when there's a big change like this.

Reply:
I have Format-on-save with Prettier. Looks like I pressed CMD+S within the file, whoops haha. Will update Web Scraper doc as well to fix some of the things you highlighted within Puppeteer Scraper one